spec/controllers/attachments_controller_spec.rb

```diff
@@ -99,4 +99,55 @@
       it_behaves_like :redirected
     end
   end
+
+  describe :show do
+    let(:file) { OpenProject::Files.create_uploaded_file name: 'foobar.txt' }
+    let(:work_package) { FactoryGirl.create :work_package, project: project }
+    let(:uploader) { nil }
+
+    let(:attachment) do
+      clazz = Class.new Attachment
+      clazz.mount_uploader :file, uploader
+
+      ##
+      # Override to_s for carrierwave to use the correct class name in the store dir.
+      def clazz.to_s
+        "attachment"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.